doc/api/path.md

```diff
@@ -68,7 +68,8 @@ changes:
 * Returns: {string}
 
 The `path.basename()` methods returns the last portion of a `path`, similar to
-the Unix `basename` command.
+the Unix `basename` command. Trailing `/` (and `\\` on Windows) characters are
+ignored.
```

Comment:
ok. but "used in return values" means what?
```
> path.win32.dirname('\foo/that/bar\\//\\//')
'\foo/that'
```
^--- clearly / and \ are both used in return values